### PR TITLE
Add scripts for restarting service and cleaning indices.

### PR DESCRIPTION
**Pull request by kriscieslak**
…ndices.

### CODE REVIEW

File: terraform/templates/restart [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+sudo systemctl stop snapshot-streaming
+
```
Comment (marcinwadon) [inline terraform/templates/restart:4]: I'd execute clean_indices here between stop and start

File: terraform/templates/clean_indices.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (marcinwadon) [inline terraform/templates/clean_indices.sh:1]: why `start` and `restart` are without `.sh` ext and `clean_indices` is with it?

**Review by marcinwadon — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -10,3 +10,4 @@ logs/
 *.tfvars
 *.tfstate
 .terraform.lock.hcl
+*.jar
```
Comment (marcinwadon) [inline .gitignore:13]: Why?

File: terraform/ec2.tf [hunk 4]
```diff
@@ -80,9 +81,23 @@ resource "aws_instance" "snapshot-streaming" {
...
+
+  provisioner "file" {
+    source = "templates/restart"
+    destination = "/home/ec2-user/snapshot-streaming/restart"
+  }
+
   provisioner "remote-exec" {
     inline = [
-      "sudo chmod 774 /home/ec2-user/snapshot-streaming/start",
+      "sudo chmod 774 /home/ec2-user/snapshot-streaming/start.sh",
```
Comment (marcinwadon) [inline terraform/ec2.tf:98]: Why?

File: .gitignore (hunk 3 above)
Comment (kriscieslak) [reply .gitignore]: Removed.

File: terraform/ec2.tf (hunk 4 above)
Comment (marcinwadon) [reply terraform/ec2.tf:98]: There were already two files, so you could add the third one in the same pattern

Comment (kriscieslak) [reply terraform/ec2.tf:98]: Which pattern do you think is better? Should I change it back?

Comment (marcinwadon) [reply terraform/ec2.tf:98]: Adding the new file I'd keep the existing pattern (and we use it across all projects and nodes)

File: terraform/templates/restart [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+cat <<EOF > ./nextOrdinal.json
+{
+  "startingOrdinal": 0,
+  "gaps": []
+}
+EOF
+
+./clean_indices.sh
```
Comment (marcinwadon) [inline terraform/templates/restart:12]: It won't work now

**Review by marcinwadon — CHANGES_REQUESTED** [review]

File: terraform/templates/snapshot-streaming.service [hunk 6]
```diff
@@ -4,7 +4,7 @@ Description=Constellation Snapshot-Streaming
 [Service]
 User=ec2-user
 WorkingDirectory=/home/ec2-user/snapshot-streaming
-ExecStart=/home/ec2-user/snapshot-streaming/start
+ExecStart=/home/ec2-user/snapshot-streaming/start.sh
```
Comment (tbekas) [inline terraform/templates/snapshot-streaming.service:7]: In `terraform/ec2.tf` in line 98 the name of the script is provided without the `.sh` extension.
```
sudo chmod 774 /home/ec2-user/snapshot-streaming/start
```

**Review by tbekas — APPROVED** [review]
LGTM

**Review by marcinwadon — APPROVED** [review]

**Pull request merged**